.github/workflows/docker_image.yaml

```
@@ -0,0 +1,22 @@
+name: Build and publish docker image to docker hub
+
+on:
+  push:
+    tags:
+    - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build and publish to registry
+        uses: elgohr/Publish-Docker-Github-Action@master
```

I would nail the version to the current latests (2.22)

```suggestion
        uses: elgohr/Publish-Docker-Github-Action@2.22
```

https://github.com/marketplace/actions/publish-docker?version=2.22